example/src/main/java/com/xero/example/RequestTokenServlet.java

```diff
@@ -16,7 +17,7 @@
  */
 public class RequestTokenServlet extends HttpServlet {
 	private static final long serialVersionUID = 1L;
-	private Config config = Config.getInstance();   
+	private Config config = JsonConfig.getInstance();
```

Comment:
This here means that even though you provide your custom `Config` implementation, it will not be used by the RequestTokenServlet. It forces the implementation to be `JsonConfig` with this line.

A better approach might be to use dependency injection. No need to use Spring or Guice but just have classes request the Config as a construction parameter instead of instanciating/fetching it themselves.